### PR TITLE
MIR-658 geographic coordinate should allow more than one point

### PR DESCRIPTION
**Pull request by AMill7**
- can now draw Box/Polygon
- update to openlayers v4

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 61220230d4a4575bf3e40dde9aa363e4514466d8! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/938-pull_request-163

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 61220230d4a4575bf3e40dde9aa363e4514466d8! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/938-pull_request-163

File: mir-module/src/main/resources/META-INF/resources/js/mir/geo-coords.js [hunk 1]
```diff
@@ -1,132 +1,202 @@
...
+                drawMap($(this));
+            }
+        });
+
+        function drawMap(btn) {
+            var input = $(btn).parent().parent().prev().find("input[name*='coordinates']")[0];
+            var typeSelect = $(btn).parent().next().find("#type")[0];
+            var mapElement = $(btn).parent().next().find(".map")[0];
+            // latitude, longitude
+            var lat = 50.930453, lon = 11.587786;
```
Comment (kkrebs) [inline mir-module/src/main/resources/META-INF/resources/js/mir/geo-coords.js]: this should be configurable properties. Is it possible to use mycore.properties here?

File: mir-module/src/main/resources/config/mir/messages_de.properties [hunk 2]
```diff
@@ -100,513 +100,518 @@ metadata.versionInfo.startLabel = (Gesamte Versionsgeschichte)
...
+mir.actions                             = Aktionen
+mir.actions.noaccess                    = Um das Dokument zu bearbeiten melden sie sich bitte an.
+mir.actions.norights                    = Sie besitzen nicht die ben\u00F6tigten Rechte um das Dokument zu bearbeiten.
+mir.admineditor                         = Bearbeiten im Admin-Editor
+mir.affiliation                         = Zugeh\u00F6rigkeit
+mir.breadcrumb.home                     = Start
+mir.cartographics.coordinates           = Geographische Koordinaten:
+mir.cartographics.coordinates.box       = Box
+mir.cartographics.coordinates.geoType   = Geometrie Typ
+mir.cartographics.coordinates.none      = Keines
```
Comment (kkrebs) [inline mir-module/src/main/resources/config/mir/messages_de.properties]: please fix:
 - Geometrie Typ -> Geometrie-Typ
 - none / Keines -> please choose / bitte wählen ("none" can be misunderstood)

**Comment by AMill7** [comment]
Hi, this is not a bug, it is a feature. :) The first point starts the polygon and the last one closes it. Otherwise you would have a linestring. [https://en.wikipedia.org/wiki/Well-known_text](url)

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 1553984a3667e609cdde987efb09887a5951ae51! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/973-pull_request-163

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 1baa2aa5fda33b0f052e2308c86d488d68a6bcc9! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/975-pull_request-163

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 1baa2aa5fda33b0f052e2308c86d488d68a6bcc9! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/975-pull_request-163

**Pull request merged**